examples/simple_hpc_pbs_vpn/readme.md

```diff
@@ -0,0 +1,64 @@
+# Build a PBS compute cluster (and set up a vpn client)
+
+Visualisation: [config.json](https://azurehpc.azureedge.net/?o=https://raw.githubusercontent.com/Azure/azurehpc/master/examples/simple_hpc_pbs_vpn/config.json)
+
+This example will create an HPC cluster ready to run with PBS Pro and set-up a vpn client (e.g for license access)
+
+## Initialise the project
+
+To start you need to copy this directory and update the `config.json`.  Azurehpc provides the `azhpc-init` command that can help here by compying the directory and substituting the unset variables.  First run with the `-s` parameter to see which variables need to be set:
+
+```
+azhpc-init -c $azhpc_dir/examples/simple_hpc_pbs_vpn -d simple_hpc_pbs_vpn -s
+```
+
+The variables can be set with the `-v` option where variables are comma separated.  The output from the previous command as a starting point.  The `-d` option is required and will create a new directory name for you.  Please update to whatever `resource_group` you would like to deploy to:
+
+```
+azhpc-init -c $azhpc_dir/examples/simple_hpc_pbs_vpn -d simple_hpc_pbs_vpn -v resource_group=azurehpc-cluster,vpn_client_lic_tar_sas_url="https://test.blob.core.windows.net/data/vpn_client_lic.tar?sp=rl&st=2017-10-30T21:35:44Z&se=2017-10-31T21:35:4Z&sv=2017-02-02&sr=b&sig=hR8VJeo8JmakqRklLpqT%2Bc1ORHfgZ90rc%3D",vpn_client_conf=cloud-license-server,vpn_address_prefix=172.10.0.0/16
+```
+
+> Note:  You can still update variables even if they are already set.  For example, in the command below we change the region to `westus2` and the SKU to `Standard_HC44rs`:
+
+```
+azhpc-init -c $azhpc_dir/examples/simple_hpc_pbs -d simple_hpc_pbs_vpn -v location=westus2,vm_type=Standard_HC44rs,resource_group=azhpc-cluster,vpn_client_lic_tar_sas_url="https://lfswe.blob.core.windows.net/data/vpn_client_lic.tar?sp=rl&st=2017-10-30T21:35:44Z&se=2017-10-31T21:35:4Z&sv=2017-02-02&sr=b&sig=hR8VJeo8JmakqRklLpqT%2Bc1ORHfgZ90rc%3D",vpn_client_conf=cloud-license-server,vpn_address_prefix=172.10.0.0/16

```

Comment:
same as above removing sasurl

Reply:
Thanks Paul, the VPN_CLIENT_LIC_TAR does not need to be tarred up to /etc/Openvpn. The files are copied to that location after extraction (I will clarify that in the readme). I will also modify to use the  sasurl template before merging (and remove the example SAS keys).